code/src/components/TodoList.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+
+import todos from '../reducers/todos'
+
+const TodoList = () => {
+  const items = useSelector((store) => store.todos.items)
+  const emptyList = items.length > 0
```

Comment:
I think this approach is great for checking empty states! I just wanted to mention that the naming of the variable can be a little confusing since emptyList is true when the list is in fact not empty and false when empty 😊